core/lib/common/worker.js

```diff
@@ -61,7 +61,7 @@ module.exports = class Worker {
 		logger = { log: console.log, status: console.log, info: console.log },
 	) {
 		this.deviceType = deviceType;
-		this.url = `${config.get('worker.url')}:${config.get('worker.port')}`;
+		this.url = `http://unix:/run/leviathan/worker.sock:`;
```

Comment:
Can we adjust this PR to make the socket vs tcp a configurable option?

Something like this?
```bash
# default
WORKER_URL = http://127.0.0.1:2000
# vs
WORKER_URL = http://unix:/run/leviathan/worker.sock
```

Reply:
Good call